### PR TITLE
Update readme with instructions for downloading demos

### PR DESCRIPTION
**Pull request by kjun9**
This adds some instructions in the top-level README for downloading our demos to run them locally.

See #1202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,14 @@ StellarGraph is built on [TensorFlow 2](https://tensorflow.org/) and its [Keras
...
 
-You can start working with the examples immediately in Google Colab or Binder by clicking the ![](https://colab.research.google.com/assets/colab-badge.svg) and ![](https://mybinder.org/badge_logo.svg) badges within each Jupyter notebook. You can also run them locally, after installing StellarGraph. One of the following might be appropriate:
+You can start working with the examples immediately in Google Colab or Binder by clicking the ![](https://colab.research.google.com/assets/colab-badge.svg) and ![](https://mybinder.org/badge_logo.svg) badges within each Jupyter notebook.
+
+Alternatively, you can run download a local copy of the demos and run them using `jupyter`. The demos can be downloaded by cloning the `master` branch of this repo, or by using the [`svn` command-line tool](https://subversion.apache.org/) to download the specific directory:
+```bash
+svn export https://github.com/stellargraph/stellargraph/branches/master/demos
```
Comment (timpitman) [inline README.md:85]: wow, it feels like about 10 years since I've used subversion - amazing that this is the easiest way!

File: README.md [hunk 2]
```diff
@@ -78,7 +78,14 @@ StellarGraph is built on [TensorFlow 2](https://tensorflow.org/) and its [Keras
...
+```
+
```
Comment (timpitman) [inline README.md:87]: For people without svn, we could also give instructions for curl, ie:
```
curl https://codeload.github.com/stellargraph/stellargraph/tar.gz/master | tar -xz --strip=1 stellargraph-master/demos
```

Comment (kjun9) [reply README.md:87]: Ah nice! I think this is much better actually, rather than "surprise" new users with an `svn` command, so I might just replace it

**Comment by kjun9** [comment]
I've left a comment https://github.com/stellargraph/stellargraph/issues/1202#issuecomment-621002025 I'll make a follow-up PR

**Pull request merged**